ledger/Cargo.toml

```diff
@@ -47,7 +47,7 @@ tempfile = "3.1.0"
 [dependencies.rocksdb]
 # Avoid the vendored bzip2 within rocksdb-sys that can cause linker conflicts
 # when also using the bzip2 crate
-version = "0.12.4"
+version = "0.13.0"
```

Comment:
Dependabot is pretty impressive, it even caught this sneaky reference to rocksdb 😍